cdi/pom.xml

```diff
@@ -88,7 +88,6 @@
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
-            <scope>provided</scope>
         </dependency>
```

Comment:
Hmm, I definitely wanted to add also this dependency, I accidentally missed it while committing. However, when I try to build the current quickstarts with latest release (6.6.0.DR1), the quickstart works fine. @slaskawi Could you please confirm that the change we were discussion (about missing dependency) will be also present in next release of 6.6?